### PR TITLE
caf: Add polichrom led glitches to know issue for NCS 2.0

### PR DESCRIPTION
**Pull request by zycz**
Add polychromatic LED  glitches caused by change in pwm_period
 to know issue for NCS 2.0

Jira: NCSDK-15707

Signed-off-by: Jan Zyczkowski <jan.zyczkowski@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/libraries/caf/leds.rst [hunk 1]
```diff
@@ -231,6 +231,8 @@ The following code snippets show examples of the DTS nodes:
 
   In this example, ``pwmleds0`` is a tri-channel color LED node, while ``pwmleds1`` is a monochromatic LED node.
   Both ``pwmleds`` nodes are pointing to the ``pwms`` properties corresponding to PWM nodes set in Example 2 in `Enabling the PWM ports`_, with the respective channel numbers.
+  It is important to set pwm period for a tri-channel color LED node to value not greater then 1 millisecond.
```
[the remark this answers is not in the document]
Comment (MarekPieta) [reply doc/nrf/libraries/caf/leds.rst:234]: Also maybe we could add this information as a note, to highlight it

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -1130,6 +1130,18 @@ NCSDK-15675: Possible advertising start failure and module state error in :ref:`
 
   **Workaround:** Manually cherry-pick and apply commit with fix from main (commit hash: ``934a25ac23125758e350b64bca23885486682109``).
 
+.. rst-class:: v2-0-0
+
+NCSDK-15707: Visual glitches for LEDs when updating the LED color
+  Due to changes in board's default DTS, the default ``pwm_period`` has been increased.
```
Comment (MarekPieta) [inline doc/nrf/known_issues.rst:1136]: `.. boards' default DTS configurations ...` (it should be plural I think)

**Review by MarekPieta — CHANGES_REQUESTED** [review]

**Review by greg-fer — APPROVED** [review]
Fully agree with @divipillai 's comments 👍

**Comment by MarekPieta** [comment]
@zycz is on vacation. I took over the PR and applied requested changes. I also added my comments. Please rereview

**Review by divipillai — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -1130,6 +1130,18 @@ NCSDK-15675: Possible advertising start failure and module state error in :ref:`
 
   **Workaround:** Manually cherry-pick and apply commit with fix from main (commit hash: ``934a25ac23125758e350b64bca23885486682109``).
 
+.. rst-class:: v2-0-0
+
+NCSDK-15707: Visual glitches when updating an RGB LED's color in :ref:`caf_leds`
+  Due to changes in the default DTS of the boards, the default PWM period has been increased.
+  The first LED channel is updated one PWM period before the remaining channels are updated.
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst:1137]:
```suggestion
  The first LED channel is updated one PWM period before the remaining channels are updated.
```
What does it mean "is updated one PWM period"? You mean "by one"?

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by pdunaj — APPROVED** [review]

**Pull request merged**